### ISSUE TITLE
Basic support for Mac OS

### ISSUE THREAD

**Issue by arcturus2** [constellation-app/constellation#21]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Running the latest version of Constellation
	* Attached the ***Support Package*** via `Help > `Support Package`
    * Checked the FAQs: https://github.com/constellation-app/constellation/faq
    * Checked that your issue isn't already filed: https://github.com/constellation-app/constellation/issues
    * Checked that there is not already a module that provides the described functionality: https://github.com/constellation-app/constellation/repositories-catalog

### Description

Provide basic support for Macs. Constellation currently does not run/isn't usable on a Mac

### Steps to Reproduce

1. Unzip Constellation on a Mac
2. Start Constellation

**Expected behaviour:** Constellation should render a graph and the varies views as expected (like on a Windows machine)

**Actual behaviour:** Constellation does not render a graph due to unsupported hardware

**Reproduces how often:** 100%

**Comment by arcturus2** [comment]
I've gotten hold of a Mac to try and progress with effort and with basic testing, Constellation does run, however there are some issues.

To get it to run I did the following:
* Downloaded the Linux version of Constellation v1.3.1 from https://constellation-app.com
* Unzip
* Deleted the jre folder
* Downloaded the MacOS JRE (with JFX) from [azul](https://www.azul.com/downloads/zulu-community/?version=java-8-lts&os=macos&architecture=x86-64-bit&package=jre-fx)
* Unzip the JRE and put that as the `jre` folder
* Ran `bin\constellation` from a terminal

The issues I've found are:
* Selecting on the graph actually makes the clicks at an offset location on the graph. This may be linked to #387 but I've tested this on NetBeans 8 with Java 8 and NetBeans 11 with Java 11 and they both have the problem.
* Opening up different views (from the Views menu) work though there seem to be graphics issues with some parts of the view not rendering and it takes a bit of fiddling to get them to show (by minimising and maximising them a few times)
